sdk/lib/collection/hash_map.dart

```diff
@@ -64,10 +64,10 @@ typedef _Hasher<K> = int Function(K object);
 /// ```dart continued
 /// planets.forEach((key, value) {
 ///   print('$key \t $value');
-///   // 5        Saturn
-///   // 4        Mars
 ///   // 3        Earth
+///   // 4        Mars
```

Comment:
Unnecessary change. You can undo this as explained above order is not guaranteed with `HashMap`.

Reply:
Done